ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.9.yaml

```diff
@@ -7,10 +7,6 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.16-openshift-4.9
-  openstack-installer:
```

Comment:
Is it being disabled just to reduce the load?

Reply:
yeah i was doing it for testing only. Ignore this change.